code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -172,6 +172,59 @@
 		if(M.stat == 2)
 			M.gib()
 
+
+/mob/living/carbon/human/proc/headbite()
```

Comment:
I am sure we already have a verb like that, and why is this better than gut?

Reply:
It takes less time than gut to kill someone. Devour head exists, though headbite acts slightly differently and is closer to what the Xenomorphs do in their own lore.